File: EaterSpring/src/main/kotlin/xyz/purema/eater/interfaces/ICustomerService.kt
package xyz.purema.eater.interfaces

import xyz.purema.eater.dtos.AuthResultDto
import xyz.purema.eater.dtos.CustomerDto
import xyz.purema.eater.dtos.form.LoginForm
import xyz.purema.eater.dtos.form.RegisterForm

interface ICustomerService {
    fun register(form: RegisterForm): AuthResultDto
    fun login(form: LoginForm): AuthResultDto
    fun getProfile(): CustomerDto
}

File: EaterNet/Eater/Interfaces/ICustomerService.cs
using Eater.Dtos;
using Eater.Dtos.Form;

namespace Eater.Interfaces
{
    public interface ICustomerService
    {
        public Task<AuthResultDto> Register(RegisterForm form);
        public Task<AuthResultDto> Login(LoginForm form);
        public Task<CustomerDto> GetProfile();
    }
}
